src/NServiceBus.AwsLambda.SQS.AcceptanceTests/NServiceBus.AwsLambda.SQS.AcceptanceTests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net7.0</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
```suggestion
    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
```

Reply:
Agreed. Also check the github action and drop 3.1 https://github.com/Particular/NServiceBus.AwsLambda.Sqs/blob/master/.github/workflows/ci.yml#L42